targets/TARGET_NUVOTON/TARGET_M2351/TARGET_NU_PFM_M2351/PinNames.h

```diff
@@ -0,0 +1,116 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017-2018 Nuvoton
+ *
```

Comment:
please add SPDX identifier to new files

Reply:
Added SPDX identifier to all M2351 files.